src/main/scala/zio/redis/api/SortedSets.scala

```diff
@@ -10,29 +10,107 @@ trait SortedSets {
 
   final val zadd = Command(
     "ZADD",
-    Tuple6(
+    Tuple5(
       StringInput,
-      OptionalInput(ZAddUpdateInput),
-      OptionalInput(ZAddChangedInput),
-      OptionalInput(ZAddIncrementInput),
-      LongInput,
-      NonEmptyList(StringInput)
+      OptionalInput(ZSetUpdateInput),
+      OptionalInput(ZSetChangedInput),
+      OptionalInput(ZSetIncrementInput),
+      NonEmptyList(ZSetMemberScoreInput)
     ),
     LongOutput
   )
 
   final val zcard   = Command("ZCARD", StringInput, LongOutput)
   final val zcount  = Command("ZCOUNT", Tuple2(StringInput, RangeInput), LongOutput)
   final val zincrby = Command("ZINCRBY", Tuple3(StringInput, LongInput, ByteInput), LongOutput)
+
+  final val zlexcount = Command("ZLEXCOUNT", Tuple2(StringInput, ZSetLexRangeInput), LongOutput)
+
+  final val zinterstore = Command(
+    "ZINTERSTORE",
+    Tuple5(
+      StringInput,
+      LongInput,
```

Comment:
`numkeys` is actually very ugly parameter. I'd improve that it in subsequent issues as we need to ensure 2 lists are of the same length. For now we can drop `numkeys` and return immediately if lenghts are not equal.